jekyll/_cci2/configuration-reference.md

```diff
@@ -666,10 +662,11 @@ See the [Windows Getting Started document]({{ site.baseurl }}/2.0/hello-world-wi
 ##### GPU executor (Linux)
 {: #gpu-executor-linux }
 
-Class                           | vCPUs | RAM | GPUs |    GPU model    | GPU Memory (GiB)
---------------------------------|-------|-----|------|-----------------|------------------
-gpu.nvidia.small<sup>(2)</sup>  |   4   | 15  | 1    | Nvidia Tesla P4 | 8
-gpu.nvidia.medium<sup>(2)</sup> |   8   | 30  | 1    | Nvidia Tesla T4 | 16
+Class                           | vCPUs | RAM | GPUs |    GPU model      | GPU Memory (GiB)
+--------------------------------|-------|-----|------|-------------------|------------------
+gpu.nvidia.small<sup>(2)</sup>  |   4   | 15  | 1    | Nvidia Tesla P4   | 8
+gpu.nvidia.medium<sup>(2)</sup> |   8   | 30  | 1    | Nvidia Tesla T4   | 16
+gpu.nvidia.large<sup>(2)</sup>  |   8   | 30  | 1    | Nvidia Tesla V100 | 
```

Comment:
what goes in this last column?

Reply:
It's 30GiB of memory on the GPU nvidia large instances

Reply:
Sorry, the above is incorrect. It is 16 GiB same as the Tesla T4. See https://cloud.google.com/compute/docs/gpus#nvidia_v100_gpus

Reply:
Thank you @SemanticallyNull!